### ISSUE TITLE
Fuji RAW support 

### ISSUE THREAD

**Issue by cannaqueers** [FSchiltz/RawParser#12]
When I load a Fuji RAW file(X-E1 in my case), the image comes out wrong, therefore I can't use the app with Fuji files. Works fine with Panasonic files on the other hand.  

I'd also like to see sliders added for clarity, color vibrancy, RGB, black & white tones, and skin tone if possible? Does the app have the ability to remove red eye? Can you add a classic chrome film simulation filter, and a Leica b&w filter? Thank you!

**Comment by FSchiltz** [comment]
For fuji raw, I need a demosaic algorithm that work for them, I'll try to write one on the weekend if I have the time. For clarity and vibrance there are no algorithms publicly avalaible so it may take some times. For the rest they are pretty hard to implement things so I cannot promise

**Comment by cannaqueers** [comment]
I'm told that draktable has good demosaic algorithm for x-trans cameras. If it's hard to implement that is understandable. But, I would think a slider for white and black tones wouldn't be too hard? As for simulations there is no way to use open source ones that are out there? It's understandable if you are busy, I am just thankful for getting a reply.

**Comment by FSchiltz** [comment]
I'm going to first add a naive algorithm for x trans and then improve it a bit. For black and white yes I'll had but I first need to rework the shadow and highlight slider. For simulation I'll look into it but I may need curve and split toning implemented first.

**Comment by cannaqueers** [comment]
Thank you. I'd look into the open source algorithm darktable as many fuji fans are a big fan of how it handles x-trans sensors. Same with photo-ninja,

**Comment by FSchiltz** [comment]
Darktable use the amaze algorithms from Rawtherapee (at least that's the only one i found in the code). They looks like there are more to fuji raw, looks like there is another bug during decompression or something because everithung is shifeted a few pixel to the right.

**Comment by cannaqueers** [comment]
Never had a problem with Darktable when I used it on my linux box with either my Fuji camera and my Panasonic camera. Just wasn't a fan of the interface. 

Unrelated note, have you seen this neat thing being shown off? http://www.theverge.com/2017/3/30/15124466/ai-photo-style-transfer-deep-neural-nets-adobe

**Comment by FSchiltz** [comment]
Yeah i saw it, there is already an app on the windows store that do something similar.

If you find some paper or pseudocode I can implement it (my laptop died so bow I have really no real time to work on the application..)
